modules/installing-ocp-agent-encrypt.adoc

```diff
@@ -8,6 +8,14 @@
 
 As an optional task, you can use this procedure to encrypt your disk or partition while installing {product-title} with the Agent-based Installer.
 
+[IMPORTANT]
+====
+In some situations, when you enable TPM disk encryption in the firmware for a bare-metal host and then boot it from an ISO that you generate with the Agent-based Installer, the cluster deployment can get stuck.
+This can happen if there are leftover TPM encryption keys from a previous installation on the host.
+For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2011634[BZ#2011634].
```

Comment:
IMO, referencing an old BZ doesn't look good here, especially if there isn't any command or workaround that could help the customers.
NOTE: that BZ was migrated to OCPBUGS-8998

Reply:
I'd recommend using `--pre-install` RHCOS boot [option](https://coreos.github.io/coreos-installer/customizing-install/#customize-options) to embed the command `tpm2_clear` if needed.